frontend/styles/index.js

```diff
@@ -0,0 +1,11 @@
+// Combines all style variables under one file: import { TYPOGRAPHY, COLORS } from './frontend/styles' 
+
+import * as colors from './color';
+import * as typography from './typography';
+import * as shapes from './shapes';
+
+export {
+  colors,
+  typography,
+  shapes,
```

Comment:
I don't think we need separate name-spaces for the values (e.g., `justWhite` isn't going to be from anywhere but `colors` and `elevHairline` or `elevGray1` isn't going to be anywhere but `shapes`), so we can just spread these:

```suggestion
  ...colors,
  ...typography,
  ...shapes,
```